recipe/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-export CFLAGS="-I${PREFIX}/include"
-export LDFLAGS="-L${PREFIX}/lib"
+autoreconf -vfi
+
```

Comment:
@mariusvniekerk it seems like the recipe works at least for linux without these changes (tested locally)...

Reply:
Lets give that a go.

Reply:
seems like it works even less....

Reply:
Sry, I also removed the pkg-config dependency when testing locally.

Reply:
Hmm, seems like I installed libGLU in docker. Maybe this is why it worked for me locally. But I guess it doesn't make sense to add libGLU to the yum_requirements...

Reply:
given that we're building that probably not :P